test/unit/routes/test-overview.js

```diff
@@ -41,8 +52,13 @@ const EXPORT_CSV = '"TeamName","Grade","Overall","Untiered","D2","D1","C2","C1",
 var app
 var route
 var getOverview
+
+var getReductionsExport
+var reductionApp
+var reductionsRoute
+
 var getSubNavStub
-var getExportCsv
+var getCaseloadExportCsv
```

Comment:
Since this is exporting all types of csv, `getExportCsv` is maybe more generic / appropriate